docs/source/AdministratorGuide/Systems/WorkloadManagement/Pilots/Pilots3.rst

```diff
@@ -16,17 +16,16 @@ Bootstrap
 Pilot3 needs a JSON file for bootstrapping. We simply call this file the ``pilot.json`` file.
 The ``pilot.json`` file is created starting from information found in the Configuration Service.
 
-The pilot wrapper (the script that starts the pilot, which is effectively equivalent to what SiteDirectors send) expects
-to find (download) such a ``pilot.json`` file from a known location, which can be for example exposed by the DIRAC
-WebApp webserver.
+The pilot wrapper (the script that starts the pilot, which is effectively equivalent to what SiteDirectors send)
+expects to find (download) such pilot.json file from a known location, or a set of them.
+Such location can be, for example, exposed via *https://* by the DIRAC WebApp webserver. Other protocols (including *file://*) are possible.
```

Comment:
```suggestion
Such a location can be, for example, exposed via *https://* by the DIRAC WebApp webserver. Other protocols (including *file://*) are possible.
```
I find this statement about the file protocol confusing. At the moment the provided pilot wrapper only takes a list of hosts, and then expects the files to be on that host, reachable via https and at a specific path. Or am I missing something?

Reply:
The pilot wrapper may find such file(s) in the local file system, e.g. on CVMFS, so `file:///cvmfs/some/path.json` (I think `urllib` and `request` handles these cases).

Reply:
And how is the pilotWrapper told to look at, e.g., cvmfs? Looking at the pilotWrapper code the only urlopen calls explicitly contain `'https://' + loc`

Reply:
Ah, you are right, I was mixing with my running test. Now, 2 options: 
- change the doc and leave the cod as is
- change the code
is the second option needed (now)?

Reply:
Change the doc.

Reply:
Done.